From 7dad9dd0b796745df3c65966b2cce3d74dc1a8ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micka=C3=ABl=20Sala=C3=BCn?= <mic@digikod.net>
Date: Fri, 1 Sep 2023 13:08:19 +0200
Subject: [PATCH 3/3] WORKSHOP: Disable jpeg to avoid build error

The API changed:

coders/jpeg.c: In function 'ReadJPEGImage':
coders/jpeg.c:1129:30: error: 'struct jpeg_decompress_struct' has no member named 'process'; did you mean 'progress'?
 1129 |   image->interlace=jpeg_info.process == JPROC_PROGRESSIVE ?
      |                              ^~~~~~~
      |                              progress
coders/jpeg.c:1129:41: error: 'JPROC_PROGRESSIVE' undeclared (first use in this function)
 1129 |   image->interlace=jpeg_info.process == JPROC_PROGRESSIVE ?
      |                                         ^~~~~~~~~~~~~~~~~
coders/jpeg.c:1129:41: note: each undeclared identifier is reported only once for each function it appears in
coders/jpeg.c:1131:32: error: 'struct jpeg_decompress_struct' has no member named 'process'; did you mean 'progress'?
 1131 |   image->compression=jpeg_info.process == JPROC_LOSSLESS ?
      |                                ^~~~~~~
      |                                progress
coders/jpeg.c:1131:43: error: 'JPROC_LOSSLESS' undeclared (first use in this function)
 1131 |   image->compression=jpeg_info.process == JPROC_LOSSLESS ?
      |                                           ^~~~~~~~~~~~~~
coders/jpeg.c: In function 'WriteJPEGImage':
coders/jpeg.c:2318:61: error: format not a string literal and no format arguments [-Werror=format-security]
 2318 |           CoderWarning,"LosslessToLossyJPEGConversion",image->filename);
      |                                                        ~~~~~^~~~~~~~~~
---
 trunk/PKGBUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index cc9e8f41bade..c712fb3d59a2 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -33,7 +33,7 @@ build() {
     --with-webp --with-gslib --with-gs-font-dir=/usr/share/fonts/Type1 \
     --with-perl --with-perl-options="INSTALLDIRS=vendor" --with-lqr --with-rsvg \
     --enable-opencl --with-openjp2 --without-gvc --without-djvu --without-autotrace \
-    --without-jbig --without-fpx --without-dps --without-fftw $EXTRAOPTS
+    --without-jbig --without-fpx --without-dps --without-fftw --without-jpeg $EXTRAOPTS
   make
 }
 
-- 
2.42.0

